docs/dev/table/sql/index.md

```diff
@@ -28,7 +28,7 @@ This page describes the SQL language supported in Flink, including Data Definiti
 
 This page lists all the supported statements supported in Flink SQL for now:
 
-- [SELECT (Queries)](queries.html)
+- [Queries](queries.html)
```

Comment:
can we revert this change? Having the keyword directly can help (`SELECT, VALUES`).